### DISCUSSION TITLE
What is testuser@proxyserver here?

### DISCUSSION THREAD

**Discussion by Tikam02** [ssh-mitm/ssh-mitm#116] [Q&A] [answered]
# What is testuser@proxyserver here?

How to connect it?

## Issue

ssh: Could not resolve hostname proxyserver: Temporary failure in name resolution

**Comment by manfred-kaiser** [comment, marked as answer]
This is only an example how to use the ssh command. 

"testuser" is the user on the remote machine and "proxyserver" is SSH-MITM.
